### ISSUE TITLE
Making use of opencv-contrib-python-headless>=4.2.0.32 and check if we need latest version of pyside 6.5.1.1

### ISSUE THREAD

**Issue by mxochicale** [SciKit-Surgery/scikit-surgeryvtk#219]
### opencv-contrib-python-headless

Making use of opencv-contrib-python-headless>=4.2.0.32 to avoid "AttributeError: module 'cv2.aruco' has no attribute 'Dictionary_get' #58" in other libraries (e.g. https://github.com/SciKit-Surgery/scikit-surgeryutils/issues/58).

[requirements.txt](https://github.com/SciKit-Surgery/scikit-surgeryvtk/blob/08cb29d2596f1325a8cac6edb9ecb9cfd4b68045/requirements.txt) for [v1.1.1](https://github.com/SciKit-Surgery/scikit-surgeryvtk/releases/tag/v1.1.1)
```
opencv-contrib-python-headless>=4.2.0.32
scikit-surgerycore>=0.1.7
scikit-surgeryimage>=0.10.1
```

### pyside
See latest versions of https://pypi.org/project/PySide6/#history
You can see tags for the changes of versoin of pyside6 https://code.qt.io/cgit/pyside/pyside-setup.git/

**Comment by mxochicale** [comment]
Note that the current package versions in the requirements are  
```
scikit-surgerycore>=0.1.7
scikit-surgeryimage>=0.10.1 
```
* Latest version is 0.6.10 https://github.com/SciKit-Surgery/scikit-surgerycore/tags
* Latest version is 0.10.1 https://github.com/SciKit-Surgery/scikit-surgeryimage/tags

**Comment by mxochicale** [comment]
Specifying `opencv-contrib-python-headless>=4.2.0.32` in requirements and python virtual environment leads to install locally and remotely the latest version of the lib:
```
opencv-contrib-python-headless-4.7.0.72 #LOCAL
opencv-contrib-python-headless-4.7.0.72 #REMOTE: 
#https://github.com/SciKit-Surgery/scikit-surgeryvtk/actions/runs/5314435124/jobs/9621648370#step:5:268
```

**Comment by mxochicale** [comment]
* Created release 2.0.1
```
git tag -a 'v2.0.1' -m "Release version v2.0.1"
git push origin v2.0.1
```
* Published pypi final release 2.0.1 https://pypi.org/project/scikit-surgeryvtk/2.0.1/
